src/app/components/download/download-dialog.component.html

```diff
@@ -115,10 +84,9 @@
       --mount dst=/output,src="$PWD",type=bind \
       --rm \
       gisaia/arlas-hits-exporter \
```

Comment:
I think we should add the versions `gisaia/arlas-hits-exporter:2` to make sure to use the right image